README.md

```diff
@@ -47,6 +47,8 @@ for *temporary* AWS credential.
 
 If you are using pre-built libraries, please specify the path of library. Take pre-build library for Mac as example, you can specify `src/resources/lib/mac` as <NativeLibraryPath>.
 
+**Note**: The demo app expects the stream you specify to exist in the `us-west-2` region.
```

Comment:
Instead of expects, can you reword to: "set to run in us-west-2 by default"? 

The demo app can be modeled to read region from environment. I will include a PR to fix that in the demo. But in the meanwhile, lets reword this

Reply:
in other words, change this to:
```
The demo app is set to run in us-west-2 by default.
```
?

Reply:
How about:
```
**Note**: Make sure the stream name that you specify exists in the region the app runs in. This demo app is set to run in us-west-2 by default.
```

Reply:
Sounds good

Reply:
done